components/common/banner.js

```diff
@@ -20,12 +20,14 @@ const Banner = ({ icon, children, onCloseClick }) => {
       borderColor='ocean'
       borderBottom='1px'
     >
-      <Icon
-        name={icon}
-        size={{ base: '2rem', md: '3rem' }}
-        marginRight={{ base: 0, md: '1.5rem' }}
-        marginBottom={{ base: '1.5rem', md: 0 }}
-      />
+      {icon === 'givingHand' && (
```

Comment:
what's going on here ? looks like a weird way to accomplish this

Reply:
Ya help me find a better way 😃